imap_processing/hit/l1a/hit_l1a.py

```diff
@@ -0,0 +1,212 @@
+"""Decommutate HIT CCSDS data and create L1a data products."""
+
+import logging
+import typing
+from collections import defaultdict
+from dataclasses import fields
+from enum import IntEnum
+from pathlib import Path
+
+import numpy as np
+import xarray as xr
+
+from imap_processing import decom, imap_module_directory, utils
+from imap_processing.cdf.global_attrs import ConstantCoordinates
+from imap_processing.cdf.utils import write_cdf
+from imap_processing.hit import hit_cdf_attrs
+from imap_processing.hit.l0.data_classes.housekeeping import Housekeeping
+
+logger = logging.getLogger(__name__)
+
+
+class HitAPID(IntEnum):
+    """
+    HIT APID Mappings.
+
+    Attributes
+    ----------
+    HIT_HSKP: int
+        Housekeeping
+    HIT_SCIENCE : int
+        Science
+    HIT_IALRT : int
+        I-ALiRT
+    """
+
+    HIT_HSKP = 1251  # Housekeeping
```

Comment:
These comments are duplicated from the docstring, remove

Reply:
Yes, these comments are redundant. Good catch 👍